app/perfsymboltable.cpp

```diff
@@ -475,14 +474,14 @@ static QFileInfo findDebugInfoFile(const QString &root, const QString &file,
 
     // try in .debug folder
     if (!folder.isEmpty()) {
-        debugLinkFile.setFile(dir, folder + QDir::separator() + QLatin1String(".debug")
-                                    + QDir::separator() + debugLinkString);
+        debugLinkFile.setFile(dir.path() + QDir::separator() + folder + QDir::separator() + QLatin1String(".debug")
```

Comment:
why this change?

Reply:
got tricked by an ui bug which let me believe that this code does not work

Reply:
If I call QFileInfo::setFile with dir is `/path/to/sysroot` and folder is `/usr/lib/` QFileInfo will point to `/usr/lib/...`